### PR TITLE
orgs_serverside_validations

### PR DESCRIPTION
**Pull request by decareano**
adding more validations (serverside) to the creation of New Organisations `~/LocalSupport/app/views/organisations/.form_html.erb.` This is an attempt to finish first PT https://www.pivotaltracker.com/n/projects/742821/stories/61986126. I will then proceed to create validations for Edit Organisations PT https://www.pivotaltracker.com/n/projects/742821/stories/113360633

@decareano I've edited to provide direct links to PT - can save valuable time :-)

### CODE REVIEW

**Comment by tansaku** [comment]
Hi @decareano note that I think finishing https://www.pivotaltracker.com/n/projects/742821/stories/61986126 is particularly challenging.  It's not clear how much business value it delivers, and see my detailed comments in my own PR https://github.com/AgileVentures/LocalSupport/pull/259

There is a complicated nest of things here - we can certainly try and navigate it together, but be prepared that this touches a lot of different parts of the system for questionable improvement of the end user experience.  Happy discuss further ...

File: app/assets/javascripts/organisations.js.coffee [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
 jQuery ->
-  $("input[type='checkbox'].publish").bootstrapSwitch(size: 'mini', onText: 'public', offText: 'private')
+  $("input[type='checkbox'].publish").bootstrapSwitch(size: 'mini', onText: 'public',offText: 'private')
```
Comment (tansaku) [inline app/assets/javascripts/organisations.js.coffee]: @decareano you've accidentally deleted a single space - please revert and watch out for that - you only want substantive changes in the PR, and this is also breaking the consistency of the code layout ...

**Comment by decareano** [comment]
Hi @tansaku, no need to spend time on https://www.pivotaltracker.com/n/projects/742821/stories/61986126. Maurouen suggested to add more validations. But I understand if it does not deliver value. I would like to work on validations on the server side though at a later time. Do you want to cancel PR #328? I can work on this one: https://www.pivotaltracker.com/n/projects/742821/stories/113360633

**Comment by tansaku** [comment]
> I understand if it does not deliver value

Not that there is no value - more that the value at this time is questionable, since it's not a client/user driven need ...

**Comment by tansaku** [comment]
just getting the flash messages consistent seems like a simpler and more straightforward task for the time being :-)

**Pull request closed without merging**